<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadEditStudentGradesPageClass</class>
 <widget class="QMainWindow" name="UploadEditStudentGradesPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UploadEditStudentGradesPage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>1031</width>
      <height>651</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Upload / Edit Student Grades:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="UploadEditIDTextField">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter Student ID :  </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Course :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxCourse">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter Grade :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="UploadEditGradeTextField">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>280</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="UploadStudentGradeBtn">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>410</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EditStudentGradeBtn">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>410</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showCourses">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>130</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Courses</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Semster :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Year :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="SemstersComboBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>200</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="yearsComboBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>230</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="BackPushBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1047</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
